server/taskflows/hpccloud/taskflow/nwchem/__init__.py

```diff
@@ -76,8 +87,6 @@ def create_geometry_symlink(task, job, cluster, fileName):
 @cumulus.taskflow.task
 def setup_input(task, *args, **kwargs):
     input_folder_id = kwargs['input']['folder']['id']
```

Comment:
So I tried testing this with the development VM and I am now getting the following error when trying to run a NWChem simulation. I added a print statement to check `kwargs` and found that it is empty. I confirmed that things are still working in master.
```
[15:26:42.209] ERROR: Exception raise by task.
  File "/usr/local/lib/python2.7/dist-packages/celery/app/trace.py", line 240, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/celery/app/trace.py", line 438, in __protected_call__
    return self.run(*args, **kwargs)
  File "cumulus/taskflow/__init__.py", line 117, in wrapped
    return func(celery_task, *args, **kwargs)
  File "/opt/hpccloud/hpccloud/server/taskflows/hpccloud/taskflow/nwchem/__init__.py", line 89, in setup_input
    input_folder_id = kwargs['input']['folder']['id']
KeyError: 'input'
```